src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1415,12 +1415,9 @@ public void onPlayerSlotChange(PlayerItemHeldEvent event) {
 		BendingPlayer bPlayer = BendingPlayer.getBendingPlayer(player);
 		int slot = event.getNewSlot() + 1;
 		
-		if (bPlayer.getAbilities().get(slot) != null ) {
-			CoreAbility ability = CoreAbility.getAbility(bPlayer.getAbilities().get(slot));
-			if (ability != null) {

```

Comment:
This check is unneeded because displayMovePreview() requires null input to distinguish empty slots